src/pipelines/sass.rs

```diff
@@ -34,7 +36,12 @@ impl Sass {
         let mut path = PathBuf::new();
         path.extend(href_attr.split('/'));
         let asset = AssetFile::new(&html_dir, path).await?;
-        Ok(Self { id, cfg, asset })
+
+        // Check if the specified SASS/SCSS file should be inlined
+        let use_inline = attrs.get(ATTR_INLINE).is_some();
+
+        // Return a new SASS/SCSS pipeline
+        Ok(Self { id, cfg, asset, use_inline })
```

Comment:
Proper punctuation at the end of these comments (thanks for proper capitalization).